src/components/grid/Row.js

```diff
@@ -27,4 +28,4 @@ const Row = React.createClass({
   }
 });
 
-module.exports = Row;
+module.exports = Radium(Row);
```

Comment:
Currently I think the radium issue is mostly around webviews and I think we're concerning ourselves to much with that sub set since it is covered by the mobile mx component library.